### DISCUSSION TITLE
Docker Desktop Crashes on CLI Click

### DISCUSSION THREAD

**Discussion by pballai-talend** [community/community#14790] [Projects and Issues]
Docker Desktop 4.7.0 (77141). Any container. Clicking the CLI option crashes app immediate. I have re-set to factory default and created 1 container with the same failed result. Host system is Window 11. WSL 2 backend

**Comment by ellis-belfer-ctr** [comment]
It doesn't crash immediately for me.  But it is crashing too much where I can't get anything done.

**Reply by abhisheksrivastava9** [reply]
same problem here in windows 10. wsl2 backend

**Reply by ellis-belfer-ctr** [reply]
I upgraded to the latest Windows release which fixed the problem.
